src/code/z_skelanime.c

```diff
@@ -719,7 +724,10 @@ Gfx* SkelAnime_DrawFlex(GlobalContext* globalCtx, void** skeleton, Vec3s* jointT
     return gfx;
 }
 
-// unused
+/**
+ * Unpacks frame data from the object into frameTable
+ * Used by the legacy animation format
+ */
```

Comment:
I wouldnt assume they are coming from an object, even though that is likely the case